.clang-format

```diff
@@ -0,0 +1,21 @@
+---
+BasedOnStyle: LLVM
+IndentWidth: 4
```

Comment:
I assume we want `UseTab: Never`, or is that implied by the base `LLVM` style?

Reply:
I think so, but it doesn't hurt to add it.